### ISSUE TITLE
Question about distillation loss implementation of kd_manager.py

### ISSUE THREAD

**Issue by czjghost** [RaptorMai/online-continual-learning#12]
Hi. Thanks for your wonderful works. BER, ASER, SCR and your survey have enlighten me a lot, and let me know the tendency about online class incremental learning. Here I want to inquire about the codes implemented in kd_manager.py, why we need to choose the logits of new classes to distill? 

I found that "End-to-End Incremental Learning" uses the logits of old classes in all samples to distill. But others, such as BiC, I only know they only used the logits of old classes, does they used all samples to distill or only used old classes' samples? 

I also want to inquire about the difference and performance of KD loss, especially for replay method, among "used the logits of old classes in all sample to distill"，"used the logits of old classes in old classes' samples to distill", " used the all logits in old classes' samples to distill" and " used the all logits in all samples to distill" ? 

I would really appreciate it if you can help me understand this. May joy and health be with you always.

**Comment by czjghost** [comment]
Hi. Thanks for your wonderful works. BER, ASER, SCR and your survey have enlighten me a lot, and let me know the tendency about online class incremental learning. Here I want to inquire about the codes implemented in kd_manager.py, why we need to choose the logits of new classes to distill? 
I found that "End-to-End Incremental Learning" uses the logits of old classes in all samples to distill. But others, such as LwF, BiC, I only know they only used the logits of old classes, does they used all samples to distill or only used old classes' samples? I also want to inquire about the difference and performance of KD loss among "used the logits of old classes in all sample to distill"，"used the logits of old classes in old classes' samples to distill", " used the all logits in old classes' samples to distill" and " used the all logits in all samples to distill" ? 
I would really appreciate it if you can help me understand this. May joy and health be with you always.

**Comment by RaptorMai** [comment]
Hi, thanks for your comment and I am glad that my work helps you. In the online setting, I remember that the way we implemented works the best. But since I did not have the knowledge distillation ablation study in my survey paper, I encourage you to fork our repo and explore the variants you mentioned in your comment to see which one works the best.

**Comment by czjghost** [comment]
> Hi, thanks for your comment and I am glad that my work helps you. In the online setting, I remember that the way we implemented works the best. But since I did not have the knowledge distillation ablation study in my survey paper, I encourage you to fork our repo and explore the variants you mentioned in your comment to see which one works the best.

Okay，Thank you very much
